### ISSUE TITLE
mistake

### ISSUE THREAD

**Issue by AroKol78** [gorhill/uBlock-for-firefox-legacy#292]
sometimes
![bugubo1](https://user-images.githubusercontent.com/70756447/107118515-d8111480-6881-11eb-871c-d642db603127.PNG)
I don't know if the errors are related to each other
UXP last build ,ubo legacy last

**Comment by JustOff** [comment]
Please update the issue with all the data requested in the [template](https://github.com/gorhill/uBlock-for-firefox-legacy/blob/master/.github/ISSUE_TEMPLATE.md).

**Comment by AroKol78** [comment]
> Please update the issue with all the data requested in the [template](https://github.com/gorhill/uBlock-for-firefox-legacy/blob/master/.github/ISSUE_TEMPLATE.md).

?

---
the first report was about my bank (logged)

I noticed errors on logged pages and during manipulation ubo

open browser (homepage - google logged)
open error console from menu
search for something in google (logged)
the "ubo" icon, press it, open the list, ...
(error reported only once as in the description)
![bugubo2](https://user-images.githubusercontent.com/70756447/107128913-c9e2e880-68c1-11eb-91db-41f9a9750968.PNG)
pages reload
```
chrome://ublock0/content/dashboard.html#settings.html
chrome://ublock0/content/dashboard.html#3p-filters.html
chrome://ublock0/content/dashboard.html#1p-filters.html
```

**Comment by AroKol78** [comment]
source code `https://github.com/MoonchildProductions/UXP/tree/master/js/xpconnect/wrappers`
this error `https://github.com/MoonchildProductions/UXP/blob/master/js/xpconnect/wrappers/XrayWrapper.cpp#L251`
"ubo" has no permission in content

**Comment by JustOff** [comment]
Let me repeat one more time, for this type of bug report you need to fill it in according to the [template](https://github.com/gorhill/uBlock-for-firefox-legacy/blob/master/.github/ISSUE_TEMPLATE.md) that automatically appears in the input field when you create a new issue.

Please update your initial message as appropriate (here's a [good example](https://github.com/gorhill/uBlock-for-firefox-legacy/issues/278)) or this issue will be closed as invalid.

**Comment by AroKol78** [comment]
I'm very sorry
`mam dość zatwirdzania ... ,wypełniania formularzy ,udowodniania że nie jestem botem, wklepywania kodów z podwójnej werifikacji,
haseł kilometrowej długości, werifikcji tosamości,......`
